src/components/settings/Recovery/index.tsx

```diff
@@ -32,7 +32,7 @@ const headCells = [
     label: (
       <>
         Recovery delay{' '}
-        <Tooltip title="You can cancel any recovery attempt when it is not needed or wanted within the delay period.">
+        <Tooltip title="You can cancel any recovery attempt when it is not needed or wanted.">
```

Comment:
We should extract these to a constant as they are the same throughout the app but I'd suggest so in a separate PR to avoid inflating the scope of this.